src/java.base/share/classes/jdk/internal/vm/TranslatedException.java

```diff
@@ -120,7 +127,8 @@ private static Throwable create(String className, String message, Throwable caus
         try {
             Class<?> cls = Class.forName(className);
```

Comment:
A question about the Class.forName usage here. Class.forName uses the defining class loader of the current class so I'm wondering if the exceptions to be decoded are always of a type defined to the boot loader? jdk.internal.vm.ci is defined to the boot loader so this code hasn't really changed, it's just "new" to java.base in this PR.